### ISSUE TITLE
CC Release Candidate - SPDX Document

### ISSUE THREAD

**Issue by klumb** [chaoss/wg-risk#106]
This issue was created to collect comments about the upcoming metrics release.

This thread is for comments about SPDX Document

GitHub location: https://github.com/chaoss/wg-risk/blob/master/metrics/SPDX_Document.md

See all released and candidate metrics at:
https://chaoss.community/metrics/

**Comment by klumb** [comment]
This metric is replacing a previously released metric - Software Bill of Materials
https://github.com/chaoss/wg-risk/blob/master/metrics/Software_Bill_of_Materials.md

Scheduled to be released for 2021-02

**Comment by sgoggins** [comment]
@klumb : This change went through a review period and discussion, and I don't think it needs to be listed as under review.

**Comment by klumb** [comment]
@sgoggins I can't find evidence that this metric has ever been released or gone through the review process. It was not included on the spreadsheet in the previous release so was not on the website. There was no announcement or issue created to collect comments. 

According to the documentation I saw, It should have been released and gone through the comment period for 2020-08 but we dropped the ball. 

This is your metric so if you tell it has been released I will defer to your judgment. However, it is too late to make it part of release 2020-08
